Makefile

```diff
@@ -51,3 +56,29 @@ test-functional:
 freeze:
 	$(call print_msg, Freezing... )
 	python setup.py build
+
+
+# Use platform flag to build multi-platform pex (requires platform wheels).
+# --platform=macosx-10.12-x86_64 --platform=linux_x86_64 --platform=win64_amd
+WHEEL_CACHE := $(PWD)/dist/wheels
+PEX_ARGS    := -v --no-pypi --cache-dir=$(WHEEL_CACHE)
```

Comment:
Just curious, what does `--no-pypi` do here? Is it because all the packages should have already been downloaded to the wheel cache so we shouldn't try to download additional dependencies?

More generally, do you think we need more description about what is going on here (`pip wheel`, `./setup.py bdist_pex`, etc.) or is it all pretty standard pex setup?